libs/ngworker/lumberjack/src/lib/logging/lumberjack-logger.builder.spec.ts

```diff
@@ -0,0 +1,133 @@
+import { TestBed } from '@angular/core/testing';
+
+import { resolveDependency } from '@internal/test-util';
+
+import { LumberjackModule } from '../configuration/lumberjack.module';
+import { LumberjackLevel } from '../logs/lumberjack-level';
+import { LumberjackLogLevel } from '../logs/lumberjack-log-level';
+import { LumberjackLog } from '../logs/lumberjack.log';
+import { Payload } from '../logs/payload';
+import { LumberjackTimeService } from '../time/lumberjack-time.service';
+
+import { LumberjackLoggerBuilder } from './lumberjack-logger.builder';
+import { LumberjackService } from './lumberjack.service';
+
+interface TestPayload extends Payload {
+  testProperty: string;
+}
+
+const lumberjackLogLevels: LumberjackLogLevel[] = [LumberjackLevel.Critical, LumberjackLevel.Error];
+
+describe(LumberjackLoggerBuilder.name, () => {
+  const testMessage = 'Test Message';
+  const fakeDate = new Date('2020-02-02T02:02:02.000Z');
+
+  let fakeTime: LumberjackTimeService;
+  let lumberjackService: LumberjackService;
+
+  beforeEach(() => {
+    TestBed.configureTestingModule({
+      imports: [LumberjackModule.forRoot()],
+    });
+
+    fakeTime = resolveDependency(LumberjackTimeService);
+    lumberjackService = (resolveDependency(LumberjackService) as unknown) as LumberjackService;
+    spyOn(lumberjackService, 'log').and.callFake(() => {});
+    spyOn(fakeTime, 'getUnixEpochTicks').and.returnValue(fakeDate.valueOf());
+  });
+
+  describe('Base builder usage', () => {
+    lumberjackLogLevels.forEach((level) =>
+      it(`should send a LumberjackLog with message "${testMessage}" and level ${level}`, () => {
```

Comment:
Don't mention the log message when it's not dynamic.

```suggestion
      it(`sends a ${level} level log`, () => {
```